mmeds/database/database.py

```diff
@@ -247,6 +247,57 @@ def set_mmeds_user(self, user):
         self.db.commit()
         return set_user
 
+    def extract_columns_from_where_query(self, where):
+        """
+        Given an SQL WHERE statement, extract only the relevant columns
+        Currently does not support the operators "BETWEEN", "LIKE", or "IN"
+        """
+        keyword_delimiter = r"\s(?:AND|and|OR|or)\s"
+        column_separator = r"(?:['\"`]?)([a-zA-Z0-9_\-]+)(?:['\"`]?\s*)(?:[<>!]?=|[<>])(?:\s*[a-zA-Z0-9'\"`]+)"
+
+        split_where = re.split(keyword_delimiter, where)
+        columns = []
+        for w in split_where:
```

Comment:
not sure if you're suggesting using re.match in place of the re.split or the re.findall, but I don't think it could work in either case. Using the split to divide the clause by ANDs and ORs, using the findall to ensure there isn't more than one (i.e. it was parsed correctly)